### PR TITLE
Add GQL status to `RuntimeUnsupportedWarning`

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 1]
```diff
@@ -1044,11 +1073,43 @@ Use a different runtime or remove the runtime option to run the query with the d
...
+----
+SHOW INDEXES YIELD *
+----
+======
+=====
 
 [#_neo_clientnotification_statement_runtimeexperimental]
 === RuntimeExperimental
 
+[WARN]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1108]: I don't know whether it is a correct decision to add a warning here. But there is no GQL status for this notification. At the same time it's in use in versions prior 5.14, so we can't just remove it

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-152.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-152.surge.sh/status-codes/5/notifications/all-notifications)

File: modules/ROOT/pages/notifications/all-notifications.adoc (hunk 1 above)
Comment (NataliaIvakina) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1108]:
```suggestion
[NOTE]
```

Comment (renetapopova) [reply modules/ROOT/pages/notifications/all-notifications.adoc:1108]: Yes, I think NOTE is better than WARNING.

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 2]
```diff
@@ -1044,11 +1073,43 @@ Use a different runtime or remove the runtime option to run the query with the d
...
-====
+
+======
+[.include-with-GQLSTATUS-code]
+======
+Query::
++
+[source,cypher]
+----
+CYPHER runtime=pipelined SHOW INDEXES YIELD *
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1084]:
```suggestion
EXPLAIN CYPHER runtime=pipelined SHOW INDEXES YIELD *
```
There is a missing EXPLAIN above as well.

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 3]
```diff
@@ -1044,11 +1073,43 @@ Use a different runtime or remove the runtime option to run the query with the d
...
+----
+
+Returned GQLSTATUS code::
+01N40
+
+Returned Status Description::
+warn: runtime unsupported.
+The query cannot be executed with `runtime=pipelined` is used.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1092]:
```suggestion
The query cannot be executed with `runtime=pipelined`, `runtime=slotted` is used.
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 4]
```diff
@@ -1024,8 +1030,31 @@ m|WARNING
...
+a|warn: runtime unsupported. The query cannot be executed with `$preparser_input1`, `preparser_input2` is used. Cause: `$msg`.
+|Severity
+m|WARNING
+|Classification
+m|UNSUPPORTED
+|===
+======
+=====
+
+==== Example of an unsupported runtime for the given query
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1051]:
```suggestion
==== Example of a runtime not supported by a Cypher command
```

File: modules/ROOT/pages/notifications/all-notifications.adoc (hunk 3 above)
Comment (renetapopova) [reply modules/ROOT/pages/notifications/all-notifications.adoc:1092]: We can also add the other example:

Parallel runtime does not support write queries
`EXPLAIN CYPHER runtime=parallel CREATE (n)`
The returned GQLSTATUS code will be the same.

I wonder if we can rename the example headings to remove the "An example of" part.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-152.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-152.surge.sh/status-codes/5/notifications/all-notifications)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-152.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-152.surge.sh/status-codes/5/notifications/all-notifications)

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 5]
```diff
@@ -1044,11 +1073,43 @@ Use a different runtime or remove the runtime option to run the query with the d
...
+EXPLAIN CYPHER runtime=pipelined SHOW INDEXES YIELD *
+----
+
+Returned GQLSTATUS code::
+01N40
+
+Returned Status Description::
+warn: runtime unsupported.
+The query cannot be executed with `runtime=pipelined`, `runtime=slotted` is used.
+Cause: `PIPELINED`.
```
Comment (Lojjs) [inline modules/ROOT/pages/notifications/all-notifications.adoc:1093]:
```suggestion
Cause: `Pipelined does not yet support the plans including `ShowIndexes`, use another runtime.`.
```

This is what I get if I actually run the query above

**Review by Lojjs — CHANGES_REQUESTED** [review]
Some things to be fixed. Also we might want to adapt this part after https://trello.com/c/5yAsUoRN/265-split-01n40-into-two-gqlstatuses is fixed

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-152.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-152.surge.sh/status-codes/5/notifications/all-notifications)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-152.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-152.surge.sh/status-codes/5/notifications/all-notifications)

**Review by Lojjs — APPROVED** [review]

**Pull request merged**